src/routes/v2/authenticatedSites/index.js

```diff
@@ -189,7 +189,7 @@ const getAuthenticatedSitesSubrouter = ({
 
   authenticatedSitesSubrouter.use(authenticationMiddleware.verifyJwt)
   authenticatedSitesSubrouter.use(attachSiteHandler)
-  authenticatedSitesSubrouter.use(authorizationMiddleware.checkIsSiteMember)
+  authenticatedSitesSubrouter.use(authorizationMiddleware.verifySiteAdmin)
```

Comment:
This should change to `verifySiteMember` instead because we still want to allow site members to access the site.

Reply:
[cef758b](https://github.com/isomerpages/isomercms-backend/pull/485/commits/cef758bc69caa8030618e55d707d165519f10652)